boards/frdm-kw41z/board.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     board_frdm-kw41z
+ * @{
+ *
+ * @file
+ * @brief       Board specific initialization for the FRDM-KW41Z
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "periph/gpio.h"
+#include "periph/rtt.h"
+
+static inline void set_lpuart_clock_source(void)
+{
+    /* Use MCGIRCLK (internal reference 4 MHz clock) */
+    SIM->SOPT2 = (SIM->SOPT2 & ~SIM_SOPT2_LPUART0SRC_MASK) | SIM_SOPT2_LPUART0SRC(3);
```

Comment:
Shouldn't this be part of the `uart_init()` function and rather then be configurable from the `periph_conf.h` if needed?

Reply:
I agree, I will try to come up with a clean enhancement to the LPUART driver

Reply:
I have added a new define for the lpuart clock sources